main.go

```diff
@@ -5,62 +5,125 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"net/http"
 	"os"
 	"regexp"
 	"strings"
-	"time"
 )
 
 func main() {
-	fmt.Printf("Please pick a word and then I will tell you how often is shows up in each of the following Shakespeare plays: The Tempest, Hamlet, King Lear, Macbeth, and Romeo and Juliet. Please pick a word...")
-	Scanner := bufio.NewScanner(os.Stdin)
-	Scanner.Scan()
-	userInputWord := Scanner.Text()
-	userInputLowercase := strings.ToLower(userInputWord)
+	log.Fatal(http.ListenAndServe(":8080", http.FileServer(http.Dir("static"))))
 
-	timeAtStart := time.Now()
+	fmt.Println("Hello and welcome to the Shakespeare Scanner.\n This is a tool which allows you to search a Shakespeare play for a word\n and it will tell you both how many times it shows up, as well as where it shows up.\n")
+	for {
+		fmt.Println("Please type the title of the play you would like to search for your word...")
+		Scanner := bufio.NewScanner(os.Stdin)
```

Comment:
Probably more idiomatic to use lowercase `scanner`

Reply:
Will edit thank you :)